AcsfToolsBackgroundTasksCommands.php

```diff
@@ -508,23 +508,21 @@ public function getBackgroundTasksSitesStatus($date = NULL, $iteration = NULL) {
     $sites = array();
 
     foreach ($acsfSites as $db => $conf) {
-      if (isset($conf['flags']['preferred_domain']) && $conf['flags']['preferred_domain'] === TRUE) {
-        $sites[$db] = array(
-          'success' => 0,
-          'error' => 0,
-          'domain' => reset($conf['domains']),
-        );
-
-        if (file_exists($gfsFlagsFolder . 'background_tasks_pending_' . $db)) {
-          $sites[$db]['flag'] = file_get_contents($gfsFlagsFolder . 'background_tasks_pending_' . $db);
-        }
+      $sites[$db] = array(
+        'success' => 0,
+        'error' => 0,
+        'domain' => reset($conf['domains']),
+      );
```

Comment:
could this benefit of some commenting to know what's going on here?

Reply:
Comments added.